.circleci/config.yml

```diff
@@ -408,6 +421,11 @@ jobs:
           key: velox-ccache-debug-{{ arch }}-{{ checksum "merge-base-date" }}
           paths:
             - .ccache/
+      - run:
+          name: "Run Unit Tests"
```

Comment:
This leg is only supposed to be for building minimal Velox. Not for running tests.

Reply:
> This leg is only supposed to be for building minimal Velox. Not for running tests.

Nice comments, I move this part to linux-adapters.